libdnf/module/ModulePackageContainer.cpp

```diff
@@ -446,7 +449,11 @@ bool ModulePackageContainer::isEnabled(const ModulePackage * module)
  */
 bool ModulePackageContainer::isDisabled(const std::string &name)
 {
-    return pImpl->persistor->getState(name) == ModuleState::DISABLED;
+    try {
+        return pImpl->persistor->getState(name) == ModuleState::DISABLED;
+    } catch (NoModuleException &) {
+        return false;
```

Comment:
I don't consider it as an error, this is a question how persistor and function is supposed to be used.

Reply:
Maybe, but what about the explanation? And unless it's obvious, I still think it should go into a code comment, because like this, at least to me the code just looks wrong.

Reply:
The original code did not throw any error but created a new entry in persistor. Then the code was change and error was catched in isEnabled but other calls were not fixed. My patch fixes also other calls.

Reply:
Ok, after more off the record discussion the API is bad (because it allows usage in which calling these methods will return ambiguous results), but we need to keep it for now for compatibility reasons and throwing exceptions already is an API breakage. We should fix it in next version of the API.